apps/to_do_gui.py

```diff
@@ -0,0 +1,215 @@
+import sqlite3 as sql
+from tkinter import *
+from tkinter import messagebox
+
+
+"""
+    To Do App with Tkinter
+"""
+connector = sql.connect("todo.db")
+cursor = connector.cursor()
+
+cursor.execute(
+    "CREATE TABLE IF NOT EXISTS Todo_List (S_NO INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, TASK TEXT, DETAIL TEXT)"
+)
+
+
+app = Tk()
+app.title("To Do App GUI")
+app.geometry("750x500")
+app.resizable(0, 0)
+
+# color and font variables
+left_frame_bg = "Gray70"
+center_frame_bg = "Gray57"
+right_frame_bg = "Gray35"
+frame_font = ("Garamond", 14)
+
+# StringVar variables
+task_name_str = StringVar()
+search_str = StringVar()
+
+# the functions
+# add a task to database
```

Comment:
Trivial: Could you move the comments inside the functions as docstrings please? Everything else looks perfect to me, thanks a lot 🙂

Reply:
I have made the required changes.